opentelemetry-sdk/src/opentelemetry/sdk/__init__.pyi

```diff
@@ -16,6 +16,3 @@
 The OpenTelemetry SDK package is an implementation of the OpenTelemetry
 API
 """
-from . import metrics, trace, util
```

Comment:
@aabmass We should be okay to remove them, the structure of the `src/` folder provided by `opentelemetry-sdk` looks like this:

```
├── src
│   ├── opentelemetry
│   │   └── sdk
│   │       ├── __init__.pyi
│   │       ├── error_handler
│   │       ├── metrics
│   │       ├── resources
│   │       ├── trace
│   │       ├── util
│   │       └── version.py
```

Which means you could always access `metrics`, `traces`, and `util` through the regular folder hierarchy at `opentelemetry.sdk.(metrics|traces|util)` when importing. Exposing them in this file doesn't appear to make them any more accessible.